macros/sql_gen/gen_joined_metrics_cte.sql

```diff
@@ -175,10 +185,11 @@
         , {{ dim }}
         {%- endfor %}
         {%- for metric_name in metric_tree.parent_set %}
-        , coalesce({{metric_name}},0) as {{metric_name}}
+        , {{metric_name}}
         {%- endfor %}  
-        {%- for metric in metric_tree.ordered_expression_set%}
-        , {{metric}}
+        {%- for metric in metric_tree.expression_set %}
+        {#- should this me ordered expression set? -#}
```

Comment:
```suggestion
        {#- should this be ordered expression set? -#}
```

Beyond the typo, also a reminder that you need to answer this note to self

Reply:
I think this is actually a note to the group! I don't think I know the answer here

Reply:
Isn't it fun when you write comments and then don't remember what you actually meant? If this was me then I suspect it was me musing on whether I should only be using the ordered expression set - but thats not an issue. As mentioned in above comment, the values in those two dicts are roughly the same and contain the same metrics (ordered contains additional info on metric depth). Expression set is cleaner because we don't have to pull just the key value